torch/distributed/rpc/internal.py

```diff
@@ -36,7 +36,7 @@ class _InternalRPCPickler:
     """
 
     def __init__(self):
-        self._dispatch_table = copyreg.dispatch_table.copy()
+        self._dispatch_table = copyreg.dispatch_table.copy()  # type: ignore[attr-defined]
```

Comment:
could you add ignore reasons to those in this file?

Reply:
Yes. Because mypy doesn't find the "dispatch_table" attribute of `copyreg`, which is a third-party library.